src/__Private/MigrationCLI.php

```diff
@@ -165,8 +166,20 @@ protected function getSupportedOptions(): vec<CLIOptions\CLIOption> {
         'Enable XHProf profiling',
         '--xhprof',
       ),
-      $this->getVerbosityOption(),
     ];
+
+    if(\Facebook\HHAST\HHVM_VERSION_ID >= 33100) {
```

Comment:
you want to check the version of HHVM that is actually running (`\HHVM_VERSION_ID`), not the version that the codegen was built against